example-client/example-client.cabal

```diff
@@ -48,3 +51,8 @@ executable example-client
                    network     >= 2.6 && < 3.2
   else
     build-depends: network     >= 2.5 && < 2.6
+
+  if flag(use-cabal-syntax)
+    build-depends: Cabal-syntax >= 3.7 && < 3.9
+  else
+    build-depends: Cabal        >= 1.12 && < 3.7
```

Comment:
I probably got confused because this section doesn't look like all others.

But this is just an example, so no need to change